### ISSUE TITLE
Why OpenSource? and Why we should care. | OSCA Mauritius - 2021/08/26

### ISSUE THREAD

**Issue by GirishMahabir** [oscafrica/chapter-leads-hub#52]
## Event information

- **Event name**: Why OpenSource? and Why we should care.
- **Event description**: Promote OpenSource
- **Event date**: 21/08/26
- **Event time**: 18:30 (GMT +4)
- **Event duration**: 1h
- **Event location**: Online
- **Estimated number of attendees**: 20-40

## How does this benefit your chapter?
This event will be targeting students and professionals.
The speaker will be the lead of Python User group in Mauritius who participated in lots of Python conferences around the world and helps a lot of people including myself get their first step into opensource.

In this workshop, he will be talking about how opensource helped him as a freelancer, how it helps students in general in their career, some of the famous open-source projects as well as projects that himself is maintaining.

## What help do you need with this event? 
Post our event on OSCA social Media.

## Checklist

**Before you post the issue**: 
- [x] You have followed the issue title format.
- [x] You have applied the correct labels (Country, Type of Event, Requests).
- [x] You have completed the [request form](https://bit.ly/osca-cl-request) (if you need sponsorship).

**Before the event**: 
- [x] Your event is approved.
- [x] Your event's RSVP page has been created.
- [x] You have received your sponsorship (if requested).

**After the event**:  
- [x] You have completed the [post-event report](https://github.com/oscafrica/chapter-leads-hub/blob/master/docs/post-event-template.md).
- [x] You have submitted all [related expense receipts](https://github.com/oscafrica/chapter-leads-hub/blob/master/docs/expenses.md#tracking-your-expenses) for this event.
- [ ] This issue is closed.

**Comment by didicodes** [comment]
Apologies for responding late, @GirishMahabir. This looks great, please go ahead and kickstart the planning and awareness proceess. 🔥

**Comment by GirishMahabir** [comment]
Thank you @didicodes 😁

**Comment by GirishMahabir** [comment]
# Post Event Report Why OpenSource? and Why we should care. | OSCA Mauritius - 2021/08/26

## Attendance 🔵
We got about 8-10 people, most of them professionals, we did expect a lot more and a lot more students as well.

## Event Venue 🟣
Jitsi -> Google Meet

We got a lot of problems with Jitsi, our tabs froze and we some couldn't see the slides.
We moved to google meet where we got some spammers that came in and started playing loud sounds. Managed to block them.

## The Good, Bad, and Ugly 🔴

The good is that we got an amazing session, super helpful and informative. 
The bad and the ugly is that we didn't get the number of attendance that we expected and the spammers we got on google meet etc was super bad.
 
## TIL (Today I Learned) ⚪️

I've learned that we now have to start advertising at least 3 weeks before the actual webinar.
I will also have to test other online conference websites to find a much more appropriate one.

## Feedback from Attendees 🟠

Moreover, we got positive feedback and learned a lot about the Mauritian mindset concerning opensource as well as tons of projects that Mauritius have on opensource.

In short, it was a real challenge and I learned a lot. Will be preparing a lot more now.

**Comment by didicodes** [comment]
Hi @GirishMahabir, great work on this event. So sorry about the challenges you encountered, and we're happy you've learned from them. Always try to put out an RSVP form and only share the link to the call to those who sign up or maybe in your Discord channel (sometimes). 

We're super happy about all that you and your team are doing; well done!

**Comment by GirishMahabir** [comment]
Hello @didicodes ,

Ok Noted. Thanks for your support again.
